README.md

```diff
@@ -1,13 +1,16 @@
 # Cluedo - Mystery Randomizer
 
-Replace this readme with your own information about your project.
+This project's goal is to create a piece inspired in the classic board game Cluedo 🕵️‍♀️. The user should be able to click on three decks of cards to randomly get: a killer, a weapon and a room. Then there should also be a button to "reveal" the mystery 🙈
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The focus on this project is to practice objects and arrays and also to manipulate the DOM with JavaScript 👩‍💻💪
 
-## The problem
+## How I built it - What I learned
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+- I've built the three card decks as arrays of objects (in the case of the killer and weapon deck) and the rooms deck is a basic array of different location names. When the user clicks on a card, a function is called which will randomly pick an element from each of the arrays: this will determine who the killer is, which weapon was used and in which room the murder took place. The mystery will always vary...
+- I've added a function to generate at random a killer's favorite weapon. This one will be chosen from a manipulated array in which I've added the killer's favorite weapon's name three extra times, so that the chances of their favorite weapon being chosen are higher. For this, I've used a basic randomizer function and the push() method 💡
```

Comment:
> three extra times

This might change in the future, best to refer to the documentation of the function itself, or a variable which controls the weight